Cython/Compiler/Code.py

```diff
@@ -24,6 +24,7 @@
 from . import Naming
 from . import Options
 from . import DebugFlags
+from . import PyrexTypes
```

Comment:
```suggestion
```

Not used?

Reply:
Right, when I started I manually checked if the return type was an instance of `PyExtensionType` before realizing that there functions that already do that.